### PR TITLE
init object-shorthand

### PR DESCRIPTION
**Pull request by ledamint**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -410,6 +410,42 @@ with (_) {
...
+
+// GOOD
+const foo = {
+    a, 
+    b,
+    c() {},
+};
+
+```
+Move shorthand properties up
```
Comment (DmitryKorlas) [inline README.md]: I guess it would be better to make it as a separate paragraph. It could be shared via separate link. What do you think?

Comment (ledamint) [reply README.md]: Yes, good point, but I think the separate paragraph is too much, I have just added a link.

File: eslint-config-ringcentral/index.js [hunk 2]
```diff
@@ -147,6 +147,9 @@ module.exports = {
         // http://eslint.org/docs/rules/semi
         semi: 'error',
 
+        // https://eslint.org/docs/rules/object-shorthand
```
Comment (DmitryKorlas) [inline eslint-config-ringcentral/index.js]: please keep key's alphabetical sort-order - move `object-shorthand` key up

File: README.md (hunk 1 above)
Comment (DmitryKorlas) [reply README.md]: ok, it's also good.

**Pull request merged**